cas-server-support-saml/src/main/resources/META-INF/spring/opensaml-config.xml

```diff
@@ -47,9 +47,11 @@
           init-method="initialize">
         <property name="builderAttributes">
             <map>
-                <!-- Sun/Oracle is the default, for Xerces, set property to org.apache.xerces.util.SecurityManager -->
+                <!-- Sun/Oracle is the default, for Xerces, set property to org.apache.xerces.util.SecurityManager
```

Comment:
Remove the above comments from here. Add the property to cas.properties, and explain what the setting does. Comment out the setting of course.